linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/ConfigurationService.scala

```diff
@@ -316,17 +316,28 @@ class ConfigurationService extends Logging {
     })
   }
 
+  def getConfigByLabelId(language: String, labelId: Integer): util.List[ConfigKeyValue] = {
+    var configs: util.List[ConfigKeyValue] = new util.ArrayList[ConfigKeyValue]()
+    if (StringUtils.isNotBlank(language) && language.equals("en")) {
+      configs = configMapper.getConfigEnKeyValueByLabelId(labelId)
+    } else {
+      configs = configMapper.getConfigKeyValueByLabelId(labelId)
+    }
+    configs
+  }
+
   def getConfigsByLabelList(
       labelList: java.util.List[Label[_]],
-      useDefaultConfig: Boolean = true
+      useDefaultConfig: Boolean = true,
+      language: String
   ): (util.List[ConfigKeyValue], util.List[ConfigKeyValue]) = {
     LabelParameterParser.labelCheck(labelList)
     val combinedLabel = combinedLabelBuilder.build("", labelList).asInstanceOf[CombinedLabelImpl]
     val label =
       labelMapper.getLabelByKeyValue(combinedLabel.getLabelKey, combinedLabel.getStringValue)
     var configs: util.List[ConfigKeyValue] = new util.ArrayList[ConfigKeyValue]()
     if (label != null && label.getId > 0) {
-      configs = configMapper.getConfigKeyValueByLabelId(label.getId)
+      configs = getConfigByLabelId(language, label.getId)
```

Comment:
The key parameters are recommended to be placed in front 
with default vaule?

 def getConfigByLabelId(labelId: Integer, language: String ="zh-CN"): util.List[ConfigKeyValue] = {
    var configs: util.List[ConfigKeyValue] = new util.ArrayList[ConfigKeyValue]()
    if ("en".equals(language)) {
      configs = configMapper.getConfigEnKeyValueByLabelId(labelId)
    } else {
      configs = configMapper.getConfigKeyValueByLabelId(labelId)
    }
    configs
  }
```

Reply:
done